slash/plugins/__init__.py

```diff
@@ -213,6 +214,11 @@ def deactivate(self, plugin):
     def _configure(self, plugin):
         cfg = plugin.get_config()
         if cfg is not None:
+            warn_deprecation('PluginInterface.get_config() is deprecated since 1.5.0. '
```

Comment:
@ayalash no need to specify the "since" version in text here. In normal deprecations this is only useful for documentation purposes, not in runtime